src/util/getChildPath.js

```diff
@@ -0,0 +1,20 @@
+import { head } from './head'
+import { getThoughts } from './getThoughts'
+import { hashThought } from './hashThought'
+import { headValue } from './headValue'
+import { rankThoughtsFirstMatch } from './rankThoughtsFirstMatch'
+import { unroot } from './unroot'
+
+export const getChildPath = (child, thoughtsRanked, showContexts) => {
```

Comment:
All functions should have a descriptive JSDOC comment.

Reply:
ok